core/src/main/java/feast/core/grpc/TrainingServiceImpl.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2018 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package feast.core.grpc;
+
+import com.google.common.base.Strings;
+import com.google.protobuf.Timestamp;
+import feast.core.TrainingServiceGrpc.TrainingServiceImplBase;
+import feast.core.TrainingServiceProto.DatasetInfo;
+import feast.core.TrainingServiceProto.FeatureSet;
+import feast.core.TrainingServiceProto.TrainingServiceTypes.CreateTrainingDatasetRequest;
+import feast.core.TrainingServiceProto.TrainingServiceTypes.CreateTrainingDatasetResponse;
+import feast.core.training.BigQueryTrainingDatasetCreator;
+import io.grpc.Status;
+import io.grpc.Status.Code;
+import io.grpc.stub.StreamObserver;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import lombok.extern.slf4j.Slf4j;
+import org.lognet.springboot.grpc.GRpcService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@Slf4j
+@GRpcService
+public class TrainingServiceImpl extends TrainingServiceImplBase {
+
```

Comment:
There is something off about saying `Training` everywhere. Shouldn't this just be `DatasetService`?